### PR TITLE
Update gsheets.md

### PR DESCRIPTION
**Pull request by rpourzand**
Gsheets fast follows

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 1]
```diff
@@ -41,21 +41,26 @@ When querying your data with Google Sheets:
...
 | Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. You may choose dimensions on their own without metrics. |
 | Time Range    | Quickly select time ranges to look at the data, which applies to the main time series for the metrics (metric time), or do more advanced filter using the "Custom" selection|
-| Granularity   | Modify the granularity of the primary time dimension.      |
 | Where         | Filter your data. This includes categorical and time filters. |
 | Order By      | Return your data order.                              |
 | Limit         | Set a limit for the rows of your output.               |
 
+Note: All metrics and dimensions have an info button which you can click into to get the description of the object as defined in your dbt project.
+
+**Modifying Time Granularity**
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
#### Modifying time granularity
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 2]
```diff
@@ -41,21 +41,26 @@ When querying your data with Google Sheets:
...
-| Granularity   | Modify the granularity of the primary time dimension.      |
 | Where         | Filter your data. This includes categorical and time filters. |
 | Order By      | Return your data order.                              |
 | Limit         | Set a limit for the rows of your output.               |
 
+Note: All metrics and dimensions have an info button which you can click into to get the description of the object as defined in your dbt project.
+
+#### Modifying time granularity
 
+When selecting time dimensions in the group by menu, you will see a list of available time granularities below the selection. By default, the lowest granularity is selected. Metric Time is the default time dimension that you can group all your metrics by.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
When selecting time dimensions in the group by menu, you will see a list of available time granularities below the selection. By default, the lowest granularity is selected. Metric time is the default time dimension that you can group all your metrics by.
```

**Review by DevonFulcher — APPROVED** [review]
Looks great!

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 3]
```diff
@@ -41,21 +41,26 @@ When querying your data with Google Sheets:
...
-| Granularity   | Modify the granularity of the primary time dimension.      |
 | Where         | Filter your data. This includes categorical and time filters. |
 | Order By      | Return your data order.                              |
 | Limit         | Set a limit for the rows of your output.               |
 
+Note: All metrics and dimensions have an info button which you can click into to get the description of the object as defined in your dbt project.
+
+#### Modifying time granularity
 
+When selecting time dimensions in the group by menu, you will see a list of available time granularities below the selection. By default, the lowest granularity is selected. Metric time is the default time dimension that you can group all your metrics by.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
When you select time dimensions in the **Group By** menu, you'll see a list of available time granularities. The lowest granularity is selected by default. Metric time is the default time dimension for grouping your metrics.
```

File: website/docs/docs/use-dbt-semantic-layer/gsheets.md [hunk 4]
```diff
@@ -41,21 +41,26 @@ When querying your data with Google Sheets:
...
 |---------------|-------------------------------------------------------|
 | Metrics       | Search and select metrics.                             |
 | Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. You may choose dimensions on their own without metrics. |
 | Time Range    | Quickly select time ranges to look at the data, which applies to the main time series for the metrics (metric time), or do more advanced filter using the "Custom" selection|
-| Granularity   | Modify the granularity of the primary time dimension.      |
 | Where         | Filter your data. This includes categorical and time filters. |
 | Order By      | Return your data order.                              |
 | Limit         | Set a limit for the rows of your output.               |
 
+Note: All metrics and dimensions have an info button which you can click into to get the description of the object as defined in your dbt project.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/gsheets.md]:
```suggestion
Note: Click the info button next to any metric or dimension to see its defined description from your dbt project.
```

**Comment by mirnawong1** [comment]
confirmed i can see the info button! shipping now and thank you s much @rpourzand and @DevonFulcher !

Screenshot 2024-05-15 at 10 05 07

**Pull request merged**